### PR TITLE
Bug 1106411 - Hide the privacy panel by default on the home screen +autoland

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1106411

This pull request tries to hide the privacy panel using a home screen configuration. This way partners can easily include it if desired.

### CODE REVIEW

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=603a82deb2fa)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a132e92c5fdc)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8b2081fc2560)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=37ec64a23073)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7d9aedf6911d)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=3977654555cf)

File: apps/verticalhome/test/unit/sources/application_test.js [hunk 1]
```diff
@@ -66,4 +67,35 @@ suite('app.js > ', function() {
     assert.ok(removeStub.calledTwice);
   });
 
+  test('blacklisted origin does not get added to grid', function() {
```
Comment (Cwiiis) [inline apps/verticalhome/test/unit/sources/application_test.js]: nit, I'd like to see a test that makes sure that the blacklist works with more than one item in it.

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=83e8e005882e)

File: apps/verticalhome/test/unit/sources/application_test.js (hunk 1 above)
Comment (KevinGrandon) [inline apps/verticalhome/test/unit/sources/application_test.js]: Another test case has been added which tests two blacklisted apps.

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ec637c6be724)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a2ec74caf790)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4612c855e7c9)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d8014ae3317d)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ec014b873ddf)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=64d48bc91f18)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d3dd536e51de)

**Comment by KevinGrandon** [comment]
Closing in favor of a new approach and refactoring configurator.

**Pull request closed without merging**